src/ocean/util/encode/Base64.d

```diff
@@ -553,6 +553,19 @@ unittest
         test!("==")(result, decode("TWE="));
         test!("!=")(result, decode(""));
     }
+
+    // don't decode short things at the end
+    {
+        auto result = cast(Const!(ubyte)[])"Maq";
+        // B should be ignored since it's missing the second byte
+        test!("==")(result, decode("TWFxB"));
```

Comment:
In the context of this PR it is obvious, but just reading the code it may be not, especially because this defines a way of handling invalid Base64 code, so what happens here isn't covered by any specification.